cime_config/buildexe

```diff
@@ -49,7 +49,14 @@ def _main_func():
         if valid:
             valid_comps.append(item)
 
-    if len(valid_comps) == 2 and  "dwav" not in case.get_value("COMP_WAV") and "dlnd" not in case.get_value("COMP_LND"):
+    datamodel_in_compset = ("datm" in case.get_value("COMP_ATM") or
```

Comment:
Please replace with:
```
datamodel_in_compset = False 
for comp in comp_classes:
  dcompname = "d"+comp.lower()
  if dcompname in case.get_value("COMP_{}".format(comp):
     datamodel_in_compset = True
```

Reply:
@jedwards4b - thanks for the improvement.